### ISSUE TITLE
Multi-label Classification

### ISSUE THREAD

**Issue by fbenites** [globality-corp/sklearn-hierarchical-classification#18]
Hi 

I am quite familiar with hierarchical classification, and I think you are doing an awesome job.

I am trying to modify to use multi-label (which sometimes is confirmed but sometimes is not). I am on the classifier.py:HierarchicalClassifier:_train_local_classifier which uses some rollup strategy, which I kind of get it, but not sure how to transform my multi-label problem to use that.
I also missing some classification problems as test (like multi-label). I would love to contribute to the project, but I need more documentation on how to proceed, or what was the idea behind thee code (the comments are a little sparse without tests).

Cheers,
Fernando

**Comment by fbenites** [comment]
Sorry just realized that the technical problem is not described:
in _train_local_classifier, I am not certain what to pass to the fit method. if X must be extended to get all the nodes, or how apply_rollup_Xy modifies X and y to be fitted. I am also trying to get the local classifier to use multi-label, but not certain how to combine the sklearn.preprocessing.MultiLabelBinarizer with the code. Any advice?

**Comment by fbenites** [comment]
Solved partially by: https://github.com/globality-corp/sklearn-hierarchical-classification/pull/20

**Comment by adamhadani** [comment]
Tracking via PR #33  going forward - will update once merged
